### PR TITLE
Aj/cold start tracing support

### PR DESCRIPTION
**Pull request by astuyve**
### What does this PR do?

Adds configuration options and documentation for Cold Start Tracing

### Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [x] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [x] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [x] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [x] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [x] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ To further configure your plugin, use the following custom parameters in your `s
 | `integrationTesting`          | Set `true` when running integration tests. This bypasses the validation of the Forwarder ARN and the addition of Datadog Monitor output links. Defaults to `false`. |
 | `logLevel`                    | The log level, set to `DEBUG` for extended logging. |
 | `skipCloudformationOutputs`   | Set to `true` if you want to skip adding Datadog Cloudformation Outputs for your stack. This is useful if you are running into the 200 output limit which can cause the stack creation to fail. |
+| `enableColdStartTracing` | Set to `false` to disable Cold Start Tracing. Used in NodeJS and Python. Defaults to `true`. |
+| `coldStartTraceMinDuration` | Sets the minimum duration for a module load event to be traced via Cold Start Tracing. Number. Defaults to `3` |
```
Comment (hghotra) [inline README.md:58]: Nit: what's the unit for this?

**Review by hghotra — APPROVED** [review]
👍🏼

File: README.md (hunk 1 above)
Comment (astuyve) [reply README.md:58]: Millis. I can add that.

[the remark this answers is not in the document]
Comment (astuyve) [reply README.md:58]: Millis, will add

Comment (duncanista) [inline README.md]: Is this in seconds? If so, should we mention it?

Comment (duncanista) [reply README.md]: Just saw other comments. Resolving.

**Review by purple4reina — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (astuyve) [reply README.md]: Millis!

**Comment by pete2786** [comment]
image

**Review by cswatt — APPROVED** [review]

**Review by joeyzhao2018 — APPROVED** [review]

**Pull request merged**